Updater/HyperDisCont.lua

```diff
@@ -8,19 +8,19 @@
 --------------------------------------------------------------------------------
 
 -- Gkyl libraries
-local Alloc = require "Lib.Alloc"
-local DataStruct = require "DataStruct"
-local Grid = require "Grid.RectCart"
-local CartField = require "DataStruct.CartField"
-local Lin = require "Lib.Linalg"
+local Alloc        = require "Lib.Alloc"
```

Comment:
I'm not approving these spacing changes to HyperDisCont and elsewhere. please see the Formatting Guidelines.